TeamTracker/app/src/main/java/com/example/teamtracker/fragments/TimerFragment.java

```diff
@@ -77,23 +76,26 @@ public void onCheckedChanged(CompoundButton compoundButton, boolean isRunning) {
     }
 
     private void showAddTaskDialog(Context context, Long duration) {
-        final EditText taskNameEditText = new EditText(context);
+        final View customLayout = getLayoutInflater().inflate(R.layout.custom_add_task_alert_dialog, null);
```

Comment:
can you help us to learn this thing? why we are passing null here as second argument.

Reply:
name it addTaskDialogLayout. customLayout - we don't understand any particular thing, right?

Reply:
![image](https://user-images.githubusercontent.com/53097587/153625066-fdaa6d69-6ba9-4663-9a15-8bc563de2390.png)

Reply:
Hope it helps you :|